src/item_menu.c

```diff
@@ -513,28 +513,28 @@ void sub_81AAC70(void)
 void GoToBagMenu(u8 bagMenuType, u8 pocketId, void ( *postExitMenuMainCallback2)())
 {
     u8 temp;
-    gUnknown_0203CE54 = AllocZeroed(sizeof(struct UnkBagStruct));
-    if (gUnknown_0203CE54 == 0)
+    gBagTaskStruct = AllocZeroed(sizeof(struct BagTaskStruct));
+    if (gBagTaskStruct == 0)
     {
         SetMainCallback2(postExitMenuMainCallback2);
     }
     else
     {
         if (bagMenuType != RETURN_LOCATION_UNCHANGED)
-            gUnknown_0203CE58.location = bagMenuType;
+            gBagPositionStruct.location = bagMenuType;
         if (postExitMenuMainCallback2)
-            gUnknown_0203CE58.bagCallback = postExitMenuMainCallback2;
+            gBagPositionStruct.bagCallback = postExitMenuMainCallback2;
         if (pocketId < POCKETS_COUNT)
-            gUnknown_0203CE58.pocket = pocketId;
-        temp = gUnknown_0203CE58.location - (POCKETS_COUNT - 1);
+            gBagPositionStruct.pocket = pocketId;
+        temp = gBagPositionStruct.location - (POCKETS_COUNT - 1);
         if (temp <= 1)
-            gUnknown_0203CE54->unk81B = 1;
-        gUnknown_0203CE54->mainCallback2 = 0;
-        gUnknown_0203CE54->unk81A = 0xFF;
-        gUnknown_0203CE54->unk81E = -1;
-        gUnknown_0203CE54->unk81F = -1;
-        memset(gUnknown_0203CE54->spriteId, 0xFF, sizeof(gUnknown_0203CE54->spriteId));
-        memset(gUnknown_0203CE54->windowPointers, 0xFF, 10);
+            gBagTaskStruct->unk81B = 1;
+        gBagTaskStruct->mainCallback2 = 0;
+        gBagTaskStruct->unk81A = 0xFF;
```

Comment:
Looks like `gBagTaskStruct` is just supposed to be called `gItemMenu` or `gBagMenu`.